libs/portfolio.tsx

```diff
@@ -0,0 +1,33 @@
+export const Portfolio = [
```

Comment:
You asked about what to call the folder this file sits in. I would think of a lib as a bunch of helper functions. This is better described as data I think. So maybe pop it under a data folder.